scm/github.go

```diff
@@ -610,6 +610,23 @@ func (s *GithubSCM) UpdateOrgMembership(ctx context.Context, opt *OrgMembershipO
 	return nil
 }
 
+// RevokeOrgMembership implements the SCM interface
+func (s *GithubSCM) RevokeOrgMembership(ctx context.Context, opt *OrgMembershipOptions) error {
```

Comment:
Suggest to rename it to `RemoveMember`

Reply:
Done.

Reply:
Might be confused with removing team members though.